experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx

```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
     ].forEach((set) => {
       render(<Subject {...set} />);
 
-      expect(screen.queryByTestId("label-not-launched")).toBeInTheDocument();
-      expect(screen.queryByTestId("label-start-date")).not.toBeInTheDocument();
-      expect(screen.queryByTestId("label-release-date")).toBeNull();
+      expect(screen.getByTestId("label-not-launched")).toBeInTheDocument();
```

Comment:
You don't need expect().toBeInTheDocument(). The call to `getByTestId()` will throw if it cant find the element.